src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -364,6 +364,16 @@ enum class SSL_DataHashAlgorithm : int64_t
 #endif
 };
 
+class SSL_DataHashSize 
```

Comment:
Also, I think it would make sense to define these values directly above the method that uses them.